project_event/views/project_project_view.xml

```diff
@@ -23,6 +23,15 @@
               name="Configuration"
               parent="project_event_menu_root"
               groups="group_project_event_manager"/>
+
```

Comment:
Make this menu invisible in the project_specific

Reply:
Yes, this should go on gitlab

Reply:
resolved all the comments

Reply:
This functionality is implemented and already merged in another branch.